src/model.rs

```diff
@@ -1391,7 +1390,7 @@ impl RelationshipType {
 	}
 
 	fn decode(value: Value) -> Result<Self> {
-		value.as_u64().and_then(RelationshipType::from_num).ok_or(Error::Decode("Expected valid RelationshipType", value))
+		value.as_u64().and_then(RelationshipType::from_num).ok_or_else(|| Error::Decode("Expected valid RelationshipType", value))
```

Comment:
`ok_or_else` over `ok_or` is not necessary as it doesn't make any functional difference (and is probably less performant? unsure if that's optimized away), but perhaps Space likes it better.